tests/study_file/test_study_file_resources.py

```diff
@@ -0,0 +1,153 @@
+import json
+
+from flask import url_for
+
+from dataservice.extensions import db
+from dataservice.api.study_file.models import StudyFile
+from dataservice.api.study.models import Study
+from tests.utils import FlaskTestCase
+
+STUDY_FILE_URL = 'api.study_files'
+STUDY_FILE_LIST_URL = 'api.study_files_list'
+
+
+class StudyFileTest(FlaskTestCase):
+    '''
```

Comment:
Small nit, but we should stay consistent with the use of double quotes for docstrings.

Reply:
yep i used study tests as reference i am gonna change for other resources as well👍